.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
I had moved this logic to the scripts mainly to keep the Travis config clean and it was common to each repo.